src/ape_ethereum/multicall/handlers.py

```diff
@@ -23,14 +25,20 @@
 
 
 class BaseMulticall(ManagerAccessMixin):
-    def __init__(self) -> None:
+    def __init__(
+        self,
+        address: AddressType = MULTICALL3_ADDRESS,
+        supported_chains: List[int] = SUPPORTED_CHAINS,
```

Comment:
Never pass a list as a default arg: https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

This could be okay here, but definitely requries comments to ensure nobody updates this constant value